man/docker-inspect.1.md

```diff
@@ -24,8 +24,85 @@ each result.
 **-f**, **--format**=""
     Format the output using the given go template.
 
+**--type**=*container*|*image*
+    Return JSON for specified type, permissible values are "image" or "container"
+
 # EXAMPLES
 
+## Getting information on an image where image name conflict with the container name.
+## e,g both image and container are named rhel7.
+
+   $ docker inspect --type=image rhel7
+   [
```

Comment:
Speaking of that, it looks like the existing output on https://github.com/docker/docker/pull/13187/files#diff-49f81c25974e6fe4a56f32e78378faa2R111 also isn't indented with 4 spaces, causing it to not show as code.

That should probably go in a separate PR, but it would be awesome if your could fix that while working on this document :-)